.changeset/six-seahorses-glow.md

```diff
@@ -0,0 +1,7 @@
+---
+'@shopify/polaris-icons': major
+'@shopify/polaris': major
```

Comment:
Sorry, now that I see this, I think for Polaris this is a minor update. For Polaris I'd say a major though. Not sure though in this case.